packages/shared/locales/en.json

```
@@ -628,6 +628,14 @@
         "ledgerMigrateIndex": {
             "title": "Do you need to migrate another Trinity account index?",
             "body": "You can also migrate later in Advanced Settings."
+        },
+        "exportTransactionHistory": {
+            "title": "Export transaction history",
+            "body": "Export your transaction history for all accounts consectutively within this profile. Exports will be in a CSV format.",
```

Whoops, just noted one more thing

```suggestion
            "body": "Export your transaction history for all accounts consecutively within this profile. Exports will be in a CSV format.",
```